Update Get-CsOnlineDirectoryTenant.md - Deprecation Info (#9001)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md

```

## DESCRIPTION

> *Note: Starting with TeamsPowerShell Module 4.0 this cmdlet will be deprecated. Users are required to use the Get-CsTenant cmdlets to view information previously present in Get-CsOnlineDirectoryTenant*

Reviewer: ```suggestion
**Note**: Starting with Teams PowerShell Module 4.0, this cmdlet is deprecated. Use the Get-CsTenant cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
```

> **Note**: Starting with Teams PowerShell Module 4.0 (available on 1st March'22), this cmdlet is deprecated. Use the Get-CsTenant cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.

Reviewer: ```suggestion
**Note**: Starting with Teams PowerShell Module 4.0 (available from March 1, 2022), this cmdlet is deprecated. Use the Get-CsTenant cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
```
